packages/react-core/src/components/Dropdown/DropdownSeparator.tsx

```diff
@@ -24,6 +27,7 @@ export const DropdownSeparator: React.FunctionComponent<SeparatorProps> = ({
         component={<Divider component={DividerVariant.div} />}
         className={className}
         role="separator"
+        {...getOUIAProps(DropdownSeparator.displayName, ouiaId, ouiaSafe)}
```

Comment:
if you want better default ids
https://github.com/patternfly/patternfly-react/pull/4769#discussion_r488157601